File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/debtor-product-category-balance.ts
import { ProductCategoryTypeEnum } from "projects/clientcare/src/app/policy-manager/shared/enums/product-category-type.enum";

export class DebtorProductCategoryBalance {
  bankAccountNumber: string;
  productCategory: ProductCategoryTypeEnum;
  balance: number;
  claimsTotal?: number = 0;
  policyId?: number;
  policyNumber?:string;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/DebtorProductCategoryBalance.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class DebtorProductCategoryBalance
    {
        public string BankAccountNumber { get; set; }
        public ProductCategoryTypeEnum ProductCategoryId { get; set; }
        public decimal Balance { get; set; }
        public decimal? ClaimsTotal { get; set; } = 0;
        public int? PolicyId { get; set; }
        public string PolicyNumber { get; set; }
    }
}
